server/store/subscription_store.go

```diff
@@ -32,6 +34,20 @@ func (s *pluginStore) LoadSubscription(subscriptionID string) (*Subscription, er
 	return &sub, nil
 }
 
+func (s *pluginStore) LoadUserSubscription(mattermostUserID string) (*Subscription, error) {
```

Comment:
hmm 2/5 `store` should avoid providing aggregate functions like this in its interface, this logic belongs in `mscalendar`, as a private method unless needed by other packages?